docs/docs/installation/installation-methods.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Installation Methods
+hide_title: true
+sidebar_position: 2
+version: 1
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Installation Methods
+
+How should you install Superset? Here's a comparison of the different options. It will help if you've first read the [Architecture](/docs/installation/architecture.mdx) page to understand Superset's different components.
```

Comment:
No need for it now, but I guess that link is where @mistercrunch 's diagram could go.

Reply:
Yep that architecture page is definitely the place for a diagram